wget.py

```diff
@@ -1,16 +1,27 @@
 #!/usr/bin/python
 
+import os
+
+def which(pgm):
+    path=os.getenv('PATH')
+    for p in path.split(os.path.pathsep):
+        p=os.path.join(p,pgm)
+        if os.path.exists(p) and os.access(p,os.X_OK):
+            return p
```

Comment:
The function can return empty, which it's not supposed to, because whatever is returned here is executed below.
Please add a default return, e.g. return '/usr/bin/wget' .